### DISCUSSION TITLE
[RELEASE] Einrichtung

### DISCUSSION THREAD

**Discussion by panicmanic2410** [nzp-team/nzportable#336] [Map Releases]
My first map, built it off from an abandoned map by Oraki5 with its source files released. It has given me some good footing with map making for NZP. Hope you guys like it!

![ss1](https://github.com/nzp-team/nzportable/assets/22651727/48ef19bf-c081-41d5-a8d8-02b3ef4a53aa)
![ss2](https://github.com/nzp-team/nzportable/assets/22651727/89e1818a-902a-43ff-87dd-d9ee88ddefc7)
![ss3](https://github.com/nzp-team/nzportable/assets/22651727/b16935a3-7a44-4a8d-8c81-b0a5753fa10e)
![ss4](https://github.com/nzp-team/nzportable/assets/22651727/3c5d502a-11db-4b9a-9ddd-900093f70d50)

Features: PaP, mystery box, two paths, all perks with their stingers set, custom ambience and new radio tune!

**Comment by MotoLegacy** [comment]
Neat map! Approved for meeting install standards, though I would suggest lowering the rate of your custom sounds to better manage file size, as well as shrinking the size of your custom menu graphic if you plan on supporting our low end devices (PSP, 3DS).

**Comment by ToasterPanic** [comment]
This map's really good! Very hectic with a full lobby. 

One question: is there any chance you could post the .map file/send it to me? I'm trying to make this work with my eight-player server but it keeps crashing. Thanks in advance!

**Comment by Jeyphr** [comment]
View my playthrough here: https://youtu.be/NX-TRWF4EoM
